test/suites/config.sh

```diff
@@ -192,13 +192,14 @@ test_config_profiles() {
   lxc profile device show onenic | grep p2p
 
   # test live-adding a nic
+  veth_host_name="veth$$"
   lxc start foo
   lxc exec foo -- cat /proc/self/mountinfo | grep -q "/mnt1.*ro,"
   ! lxc config show foo | grep -q "raw.lxc" || false
   lxc config show foo --expanded | grep -q "raw.lxc"
   ! lxc config show foo | grep -v "volatile.eth0" | grep -q "eth0" || false
   lxc config show foo --expanded | grep -v "volatile.eth0" | grep -q "eth0"
-  lxc config device add foo eth2 nic nictype=p2p name=eth10
+  lxc config device add foo eth2 nic nictype=p2p name=eth10 host_name="${veth_host_name}"
```

Comment:
This should be its own commit I think as that would need backporting along with the NetworkUp hook.

Reply:
ok will do @stgraber. Are you happy (perhaps wrong word), that the network up hook will break hotplugging p2p/bridge devices when kernel doesn't have the `netnsid_getifaddrs` kernel feature? This is because in the past, when a veth device was hotplugged, it's tc limits were not applied (and nor will its static routes be applied). 

In principle, we could only block the the hotplug from working if one or more of limits/routes were defined on the nic, and allow a simple device that doesn't need host side setup to complete on earlier kernels.

Reply:
@stgraber this is now done

Reply:
@stgraber as discussed the check for network host side keys has been added so that you can still hotplug a nic on an older kernel if you are not using those features. This was not needed for the limits only feature because in the current stable branch the limits keys are not inspected/applied when hotplugging a new nic anyway.